pkg/agent/flowexporter/exporter/exporter_perf_test.go

```diff
@@ -160,7 +161,12 @@ func setupExporter(isConntrackConn bool) (*FlowExporter, error) {
 		IdleFlowTimeout:        testIdleFlowTimeout,
 		StaleConnectionTimeout: 1,
 		PollInterval:           1}
-	exp, _ := NewFlowExporter(nil, nil, nil, nil, config.TrafficEncapModeEncap, nil, nil, nil, nil, false, nil, o)
+	_, podIPv4CIDR, _ := net.ParseCIDR("100.100.100.0/24")
```

Comment:
This CIDR doesn't match the IP addresses you generate in `addConns`. Is that ok?

Reply:
I think it's ok. Both the CIDR and the src/dst IP addresses of added conns can be arbitrary values. They are just placeholder, in my understanding.

I understand the podIPv4CIDR can be random value as long as it's not `nil`, so we ensure [v4Enabled](https://github.com/antrea-io/antrea/blob/e86d5e9637477a2a924c1e6eeb8a4b6d758d563d/pkg/agent/flowexporter/exporter/exporter.go#L165) is true. Before the flowexporter initialization PR changes, we are directly passing (v4Enabled, v6Enabled)  = (true, false) as the parameters.

Reply:
I think @wenyingd is changing how we determine whether v4 / v6 are enabled (https://github.com/antrea-io/antrea/pull/2974) so this code may change.
But I think we should use a matching CIDR to avoid confusion (e.g. `192.168/16`) and maybe a add a comment to indicate that this is only actually used to determine whether the cluster supports v4.

Reply:
The PR didn't change the flowexporter part. If we want to make the flowexporter initalization consistent with the PR, we can pass `v4Enabled`, `v6Enabled` as two parameters of `NewFlowExporter`, instead of determining it within the method like what we're currently doing. 

Previously in my initialization PR you suggested to determine it within the method, in order to reduce the number of parameters. What do you think now?

Reply:
> Previously in my initialization PR you suggested to determine it within the method, in order to reduce the number of parameters. What do you think now?

yes I guess we can revert that part, it's also in line with Wenying's change I think